packages/apps/api/src/app.ts

```diff
@@ -16,7 +16,6 @@ import { errorHandler } from './handlers/errorHandler';
 import { EmailClient } from './utils/EmailClient';
 
 export type Clients = {
-  sierra: SierraClient;
```

Comment:
Don't forget to remove your SierraClient import!

Reply:
oops

Reply:
delete even more code!!!!!